interface/wx/propgrid/manager.h

```diff
@@ -142,8 +143,8 @@ class WXDLLIMPEXP_PROPGRID wxPropertyGridPage : public wxEvtHandler,
 
     wxPropertyGridManager inherits from wxPropertyGridInterface, and as such
     it has most property manipulation functions. However, only some of them affect
-    properties on all pages (e.g. GetPropertyByName() and ExpandAll()), while some
-    (e.g. Append()) only apply to the currently selected page.
+    properties on all pages (eg. GetPropertyByName() and ExpandAll()), while some
```

Comment:
This is pretty minor, of course, but we use almost exclusively `e.g.` rather than `eg.` in the docs:
```
% g grep -F 'eg.' interface|wc -l
6
% g grep -F 'e.g.' interface|wc -l
393
```
So I'd undo this replacement to avoid decreasing consistency.